cypress/integration/searchTest.js

```diff
@@ -0,0 +1,17 @@
+import HomePage from '../../src/pages/homePage'
+import ShopPage from '../../src/pages/shopPage'
+import * as shopData from '../../testData/shopData'
+
+describe("Perform search for iPhone11 2019 16GB metall", ()=>{
+    
+    it("Search for iPhone11", ()=>{
+        HomePage.openUrl(shopData.shopPhonesUrl)
+        ShopPage.clickAdvancedSearch()
+                .selectMaterial(ShopPage.metallMaterialCheckbox)
+        ShopPage.selectYear(ShopPage.year2019Checkbox)
+        ShopPage.selectBrand(ShopPage.appleBrand)
```

Comment:
Please implement `selectMaterial` and `selectYear` methods in the same way like other methods (e.g. `clickAdvancedSearch`).
Then you will chain all methods and will remove `ShopPage` duplication.